[2025-03-17T19:17:34.662+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T19:17:34.770+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_a_db manual__2025-03-17T19:17:29.131919+00:00 [queued]>
[2025-03-17T19:17:34.881+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_a_db manual__2025-03-17T19:17:29.131919+00:00 [queued]>
[2025-03-17T19:17:34.883+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T19:17:35.021+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.events_project_a_db> on 2025-03-17 19:17:29.131919+00:00
[2025-03-17T19:17:35.063+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2697) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T19:17:35.133+0000] {standard_task_runner.py:72} INFO - Started process 2700 to run task
[2025-03-17T19:17:35.152+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.events_project_a_db', 'manual__2025-03-17T19:17:29.131919+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpo1j1van4']
[2025-03-17T19:17:35.162+0000] {standard_task_runner.py:105} INFO - Job 188: Subtask load_tables.events_project_a_db
[2025-03-17T19:17:35.710+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.events_project_a_db manual__2025-03-17T19:17:29.131919+00:00 [running]> on host c032966e105e
[2025-03-17T19:17:36.532+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.events_project_a_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T19:17:29.131919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T19:17:29.131919+00:00'
[2025-03-17T19:17:36.541+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T19:17:36.691+0000] {sql.py:278} INFO - Executing: select load_tables('events_project_a_db', 'events');
[2025-03-17T19:17:36.771+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:17:36.833+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:17:36.870+0000] {sql.py:553} INFO - Running statement: select load_tables('events_project_a_db', 'events');, parameters: None
[2025-03-17T19:17:36.972+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T19:17:37.241+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T19:17:37.242+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.events_project_a_db, run_id=manual__2025-03-17T19:17:29.131919+00:00, execution_date=20250317T191729, start_date=20250317T191734, end_date=20250317T191737
[2025-03-17T19:17:37.483+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T19:17:37.802+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
